workflow/Snakefile

```diff
@@ -43,6 +46,14 @@ if "sbs_process" in config:
     ALL_TARGETS += SBS_PROCESS_TARGETS_ALL
 
 
+if "phenotype_process" in config:
+
+    include: "targets/phenotype_process.smk"
+    include: "rules/phenotype_process.smk"
+
+    ALL_TARGETS += PHENOTYPE_PROCESS_TARGETS_ALL
+
+
 # Define the target files for the workflow
```

Comment:
Does this also show up in RED (as if it is a syntax error) in your vscode when you have this file open? It still works, but this is a bit frustrating.

Reply:
No it does not. I think this must be a VSCode configuration thing :(